plugins/hls-hlint-plugin/src/Ide/Plugin/Hlint.hs

```diff
@@ -284,16 +283,26 @@ getIdeas nfp = do
                      Just <$> liftIO (parseModuleEx flags' fp contents')
 
         setExtensions flags = do
-          hlintExts <- getExtensions flags nfp
+          hlintExts <- getExtensions nfp
           debugm $ "hlint:getIdeas:setExtensions:" ++ show hlintExts
           return $ flags { enabledExtensions = hlintExts }
 
-getExtensions :: ParseFlags -> NormalizedFilePath -> Action [Extension]
-getExtensions pflags nfp = do
+-- Gets extensions from ModSummary dynflags for the file.
```

Comment:
great!

Reply:
Argh, there's a somewhat important typo. It's supposed to be, "this is used when HLINT_ON_GHC_LIB is defined" instead of "is not". Gonna add it to the `features.md` known limitations PR.